gremlin-language/src/main/antlr4/Gremlin.g4

```diff
@@ -592,12 +592,16 @@ traversalMethod_properties
 traversalMethod_property
 	: 'property' LPAREN traversalCardinality COMMA genericLiteral COMMA genericLiteral (COMMA genericLiteralList)? RPAREN #traversalMethod_property_Cardinality_Object_Object_Object
 	| 'property' LPAREN genericLiteral COMMA genericLiteral (COMMA genericLiteralList)? RPAREN #traversalMethod_property_Object_Object_Object
+	| 'property' LPAREN genericLiteralMap? RPAREN # traversalMethod_property_Object
```

Comment:
should there be a question mark here? doesn't that make the argument optional so that it will also match `property()`? similar sort of question in the definition below....

i forget how antrl matches but i think it's in the order of definition. perhaps it would then be best to put these more specific sorts of definitions above the other two. it doesn't look like the conflict but in my mind we should go more specific to more general (genericLiteralMap -> genericLiteral).